gnomad_hail/resources/resource_utils.py

```diff
@@ -323,7 +323,7 @@ class VersionedPedigreeResource(BaseVersionedResource, PedigreeResource):
     """
     Versioned Pedigree resource
 
-    The attributes (path, import_sources and import_fun of the versioned resource are those of the default version of the resource.
+    The attributes (path, import_args and import_fun of the versioned resource are those of the default version of the resource.
```

Comment:
import_fun => import_func

Also, the other versioned resource classes still refer to source_path instead of import_args.